build/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+HOST_DIR=$(cd $(dirname "$0")/../; pwd)
+CONTAINER_DIR="/opt/rook.github.io"
+
+# install node dependencies through container
+echo "pulling npm dependencies..."
+docker run -t --user 1000 -w $CONTAINER_DIR -v $HOST_DIR:$CONTAINER_DIR node:8-alpine npm install
```

Comment:
The `package.json` in the repo root `rook.github.io` contains the definitions for which dependencies to pull, the entire repo is linked to `/opt/rook.github.io`.. running `npm install` looks at `package.json` and then pulls the code to `node_modules` in the root dir, so the next time the container is run the dependencies are present and mapped.

I had run into issues chaining commands with `docker run ... npm install && node prepare.js`